changelog.txt

```diff
@@ -9,6 +13,7 @@ Version 3.4.4
 - [PATCH] Only hit cache once when loading x-tenant idtokens (#1385)
 - [PATCH] Disregard pageload errors for the non-primary frame during interactive auth (#1357)
 - [PATCH] Avoid unnecessary BAM-cache reload (#1439, cherry-picked from #1426)
+- [PATCH] Concurrency fix: revert thread pool construction changes in CommandDispatcher (#1434)
```

Comment:
Should this be in this PR?

Reply:
Technically, it could be a separate cherry-pick of https://github.com/AzureAD/microsoft-authentication-library-common-for-android/commit/aeac34cd5d448f29f058905f5b18ee0a117a1ba3 but I am sort of cutting a corner by including it in this PR

Reply:
If this urks you, I can make a separate 1 liner PR for this

Reply:
OK - did a separate cherry-pick. Will also update the PR description & title of this to reflect this change